lms/djangoapps/support/static/support/jsx/entitlements/data/api/endpoints.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
```

Comment:
Is there any reason not to use a named `export`?

```javascript
export {
  entitlementList: 'whatever',
  inTheFutureSomeEndpoint: 'moar whatever',
};
```

it's going to cause an eslint error (maybe) if there's only one exported property, but I think it's fine to disable for now.

Reply:
[This article discusses the difference between `CommonJS` and `Babel` modules](http://2ality.com/2015/12/babel-commonjs.html).

Reply:
Confirmed with @cpennington that we support `Babel` and `CommonJS` `import` and `export`ing in `edx-platform`.